ramses_rf/protocol/command.py

```diff
@@ -389,13 +391,13 @@ def set_fan_rate(
         #  I 218 --:------ --:------ 39:159057 22F1 003 000204  # low
 
         # Type 2: 21:038634 18:126620 --:------ (less common)
-        #  - are cast as a triplet, 0.085s apart, without a seqn (---)
+        #  - are cast as a triplet, 0.085s apart, with a no seqn (---)
```

Comment:
That doesn't seem right... "with a no seqn".

Reply:
You are not looking at the latest code. @janvkem needs to rebase.
```python
        # Type 2: 21:038634 18:126620 --:------ (less common)
        #  - are cast as a triplet, 0.085s apart, without a seqn (---)
        #  - only payloads seen: '000.0[47A]'
        #  I --- 21:038634 18:126620 --:------ 22F1 003 000507
```

Reply:
Yes sorry, that is my mistake. Still new to using github and have to find a good workflow. I'm now committing by hand via the web interface which does not work that well. I will pay a bit more attention to my next PRs